charts/warpstream-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: warpstream-agent
 description: WarpStream Agent for Kubernetes.
 type: application
-version: 0.10.18
+version: 0.10.19
```

Comment:
I would at least bump the minor version because the new resources request will break the chart for users that don't have a node with 8Gi of memory in their cluster. I will at least add a note informing the users of that change.

Reply:
and by default we deploy 3 replicas so they will actually need 8 Gi of memory 3 times.